.github/workflows/release.yml

```diff
@@ -1,100 +1,77 @@
+                  fingerprint: ${{ env.GPG_FINGERPRINT }}
 
-      - name: Release
-        env:
-          GITHUB_TOKEN: ${{ secrets.REPOSITORY_PUSH_ACCESS }}
-        run: npm exec semantic-release
+            - name: Release
+              env:
+                  GITHUB_TOKEN: ${{ secrets.REPOSITORY_PUSH_ACCESS }}
+              run: npm exec semantic-release
 
-      - name: Setup QEMU
-        uses: docker/setup-qemu-action@v3
+            - name: Set Portainer stack webhook URL based on branch
+              run: |
+                  if [[ "${{ github.ref }}" == "refs/heads/main" ]]; then
+                    PORTAINER_WEBHOOK_URL=${{ secrets.PORTAINER_WEBHOOK_MAIN_URL }}
+                  else
+                    PORTAINER_WEBHOOK_URL=${{ secrets.PORTAINER_WEBHOOK_DEV_URL }}
+                  fi
+                  echo "PORTAINER_WEBHOOK_URL=$PORTAINER_WEBHOOK_URL" >> $GITHUB_ENV
+            - name: Trigger Portainer stack update
+              uses: newarifrh/portainer-service-webhook@v1
+              with:
+                  webhook_url: ${{ env.PORTAINER_WEBHOOK_URL }}
 
-      - name: Extract metadata for the Docker image
-        id: metadata
-        uses: docker/metadata-action@v5
-        with:
-          images: ghcr.io/${{ env.IMAGE_NAME }}
-          flavor: |
-            latest=${{ startsWith(github.ref, 'refs/heads/main') }}
-            suffix=-${{ github.sha }}
-
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@v3
-        with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Build and push Docker image
-        id: build
-        uses: docker/build-push-action@v5
-        with:
-          platforms: linux/amd64,linux/arm64
-          cache-from: type=gha
-          cache-to: type=gha,mode=max,ignore-error=true
-          push: true
-          tags: ${{ steps.metadata.outputs.tags }}
-          labels: ${{ steps.metadata.outputs.labels }}
-          build-args: |
-            GITHUB_ACTOR=${{ github.actor }}
-            GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
-
-      - name: Set Portainer stack webhook URL based on branch
-        run: |
-          if [[ "${{ github.ref }}" == "refs/heads/main" ]]; then
-            PORTAINER_WEBHOOK_URL=${{ secrets.PORTAINER_WEBHOOK_MAIN_URL }}
-          else
-            PORTAINER_WEBHOOK_URL=${{ secrets.PORTAINER_WEBHOOK_DEV_URL }}
-          fi
-          echo "PORTAINER_WEBHOOK_URL=$PORTAINER_WEBHOOK_URL" >> $GITHUB_ENV
-
-      - name: Trigger Portainer stack update
-        uses: newarifrh/portainer-service-webhook@v1
-        with:
-          webhook_url: ${{ env.PORTAINER_WEBHOOK_URL }}
+            - name: Purge outdated images
+              uses: snok/container-retention-policy@v3.0.0
+              with:
+                  account: revanced
+                  token: ${{ secrets.GITHUB_TOKEN }}
+                  image-names: "revanced-api"
```

Comment:
Where is the image name specified?

Reply:
```suggestion
                  image-names: "*"
```

Reply:
Changing it to * will purge everything here
https://github.com/orgs/ReVanced/packages